### PR TITLE
Test: Updated default kubernetes version to 1.9

### LINKED ISSUE #2948: Set default version of kubernetes to be tested on all PRs to 1.9
Set kubernetes 1.9 to be the default.

and `1.6`, `1.7`, `1.8` and `1.10 beta`, `1.11 alpha` - for the missed k8s versions

### PR DESCRIPTION
**Pull request by eloycoto**
Updated default kubernetes version to 1.9 in test
Added kubernetes 1.8 in test-missed-kubernetes

Fix #2948

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

**Review by aanm — APPROVED** [review]

**Review by ianvernon — APPROVED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

File: ginkgo.Jenkinsfile [hunk 1]
```diff
@@ -133,6 +81,8 @@ pipeline {
     post {
         always {
             cleanWs()
+            sh 'cd ${TESTDIR}/test/; K8S_VERSION=1.9 vagrant destroy -f || true'
```
Comment (ianvernon) [inline ginkgo.Jenkinsfile]: These commands need to be before `cleanWs()`

**Review by ianvernon — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**